### DISCUSSION TITLE
Inquiry: Marimo Server & Notebook Management

### DISCUSSION THREAD

**Discussion by stefanbringuier** [marimo-team/marimo#648] [General]
Specifically comparing its server and notebook instance management to Jupyter's, I was wondering if there is a way or a plan to have something similar.

In Jupyter, we can spin up a server on a port to manage multiple notebooks. Is there an equivalent in Marimo, or does it architecturally require separate server instances for each notebook?

I'm asking because It becomes cumbersome to manage, for example, 10 separate Marimo server instances on different ports for multiple notebooks, especially when you want to run it on a remote server.

**Comment by brunns** [comment]
I think that's what's being discussed in #601.

**Reply by stefanbringuier** [reply]
Yep, looks like. Will close this discussion, since there's a feature request in issues.
